chcemvediet/apps/inforequests/templates/admin/inforequests/action/delete_confirmation.html

```diff
@@ -14,27 +14,32 @@
         <ul>{{ inbound|unordered_list }}</ul>
       {% endif %}
     {% endafter %}
-    {% if not object.is_last_action %}
-      {% before path=".//form" %}
+    {% if snoozed_actions %}
```

Comment:
@martinmacko47 Teraz su vlastne obe templaty pre delete_confirmation a delete_selected identicke (* az na extend). Este si pockam, do jedneho suboru to dam po zapracovani ostatnych `inforequest.models`.

Reply:
Dobre zapracujem to hned po mergnuti tohoto PR, a Branch bulk delete PR.